homeassistant/components/fritz/__init__.py

```diff
@@ -34,7 +34,7 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         await avm_wrapper.async_setup(entry.options)
     except FritzSecurityError as ex:
         raise ConfigEntryAuthFailed from ex
-    except (FritzConnectionException, FritzResourceError) as ex:
+    except (FritzConnectionException, FritzServiceError) as ex:
```

Comment:
While at it, why not use a common block:

https://github.com/home-assistant/core/blob/dev/homeassistant/components/fritz/common.py#L520-L549

?

Reply:
I'm not sure, if I get it correct? the linked block is not suitable here

Reply:
as discussed on discord, the exceptions are now unified into one constant